### PR TITLE
[Design2-112] DSCO - Chips - Add gray color, thick/thin text types

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design2-112] DSCO - Chips - Add gray color, thick/thin text types

* Added `gray` color
* Added `thick` and `thin` options (chips text type)
* Update vertical alignment of chips text

Знімок екрана 2024-05-22 о 20 11 14
Знімок екрана 2024-05-22 о 20 11 18

Updated vertical alignment:
image

## Links
- jira ticket: [Design2-112](https://codedotorg.atlassian.net/browse/DESIGN2-112)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/componentLibrary/chips/chip.module.scss [hunk 1]
```diff
@@ -98,14 +98,39 @@
   }
 }
 
+// Chips Colors
+.chips-black {
+  fieldset {
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/chips/chip.module.scss:103]: TIL! @hannahbergam [check this out](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset)! Could come in handy for sign up flow 🙌

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

**Review by kelbyhawn — APPROVED** [review]
Looks good! 🎉 The vertical alignment in the first two screenshots still looks a bit off, but I checked this in Storybook and it looks good there: 

| Black and gray | Thick and thin |
| ---- | ---- |
| Black and gray | Thick and thin |

**Comment by levadadenys** [comment]
> Looks good! 🎉 The vertical alignment in the first two screenshots still looks a bit off, but I checked this in Storybook and it looks good there:

Correct, thanks for spotting!
Forgot to update these two screenshot with vertical alignment update. Updated the screenshots.

File: apps/src/componentLibrary/chips/chip.module.scss (hunk 1 above)
Comment (hannahbergam) [reply apps/src/componentLibrary/chips/chip.module.scss:103]: oooo fun! Thank you!

**Pull request merged**